### DISCUSSION TITLE
Add Opcache note in cache section

### DISCUSSION THREAD

**Discussion by bigb06** [smarty-php/smarty#1057] [General]
It can be useful to explain that if file cache is enabled, it can waste quickly opcache memory.
Cache folder may be exclude via [opcache.blacklist_filename](https://www.php.net/manual/fr/opcache.configuration.php#ini.opcache.blacklist-filename)

https://smarty-php.github.io/smarty/5.x/api/caching/basics/

**Comment by wisskid** [comment]
Is this still a thing after #1007 was fixed in v5.4.0?
